draft-iab-protocol-maintenance.md

```diff
@@ -323,11 +323,12 @@ From observing widely deployed protocols, it appears there are two stable points
 on the spectrum between being strict versus permissive in the presence of
 protocol errors:
 
-* If implementations predominantly enforce strict compliance with
+* If continually updating implementations predominantly enforce strict compliance with
```

Comment:
I don't think that we want this change.  It dilutes the message too much.

Reply:
I agree. We already have this scope in the Applicability section.